more clearly explain ongoing signaling, and use 'network to host' and… (#22)
danwing/cidfi: draft-wing-cidfi.md

The metadata exchanged over this channel is described in {{metadata-exchanged}}.

# Ongoing Signaling {#ongoing}

> Due to environmental changes on wireless networks or other user's

Reviewer: ```suggestion
Typically, due to environmental changes on wireless networks or other user's
```

Reviewer: To make sure these are only examples

traffic patterns, a particular flow may be able to operate faster or

> might need to operate slower.  The affected CIDFI Network Element SHOULD signal

Reviewer: ```suggestion
might need to operate slower.  The relevant CNE SHOULD signal
```

such conditions to the client ({{network-to-host}}), which can then
relay that information to the server using either CIDFI or via its
application.

> For example, a client streaming video might be retrieving low quality

Reviewer: ```suggestion
For example, a  streaming video client might be retrieving low quality
```

> video because one of their CIDFI network elements indicated constrained

Reviewer: ```suggestion
video because one of their invoked CNEs indicated constrained
```

bandwidth.  Later, after moving closer to an antenna, more bandwidth is

> available which is signaled by the CIDFI network element to the client.

Reviewer: ```suggestion
available which is signaled by the CNE to the client.
```

The client uses that signal to now request higher-quality video from
the server.

Similarly, the CIDFI client may begin receiving traffic

> with different characteristics which SHOULD be signaled to the CIDFI

Reviewer: ```suggestion
with different characteristics which might be be signaled to the CIDFI
```

Reviewer: This is actually up to the application client whether to ask for differentiated behaviors for these streams.
